HPXMLtoOpenStudio/measure.rb

```diff
@@ -1932,18 +1934,22 @@ def self.add_pools_and_hot_tubs(runner, model, spaces)
     @hpxml.pools.each do |pool|
       next if pool.type == HPXML::TypeNone
 
+      pool.heater_usage_multiplier *= @hpxml.header.additional_properties.misc_loads_adj_factor if pool.heater_type != HPXML::TypeNone
```

Comment:
Doesn't seem like `if pool.heater_type != HPXML::TypeNone` is needed since this code is right below `next if pool.type == HPXML::TypeNone`.

Reply:
Pool can, and cannot, have a heater, right? (`type` vs `heater_type`)

Reply:
Ah, you are right, I didn't notice that subtlety. Thanks.